src/Features/Core/Portable/Microsoft.CodeAnalysis.Features.csproj

```diff
@@ -130,6 +130,7 @@
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsVersion)" />
     <PackageReference Include="Microsoft.DiaSymReader" Version="$(MicrosoftDiaSymReaderVersion)" />
     <PackageReference Include="Microsoft.CodeAnalysis.AnalyzerUtilities" Version="$(MicrosoftCodeAnalysisAnalyzerUtilitiesVersion)" />
+    <PackageReference Include="Microsoft.SourceLink.Tools" Version="$(MicrosoftSourceLinkToolsVersion)" />
```

Comment:
This is a source package, and I've verified F5ing with VS Preview works totally fine, so I think this is okay to target main?

Reply:
Anyone know how to ignore formatting warnings/errors from this? Adding `NoWarn` to the package reference didn't work

Reply:
Not sure if we'll need a validation build or handholding for new dependencies through RPS

Reply:
A source package means that adding this reference just copies https://github.com/dotnet/sourcelink/blob/main/src/SourceLink.Tools/SourceLinkMap.cs into the obj directory, and adds it to the project, so shouldn't be any new DLL loads or anything.